wicket-core/src/main/java/org/apache/wicket/core/util/string/JavaScriptUtils.java

```diff
@@ -152,7 +153,9 @@ public static void writeJavaScriptUrl(final Response response, final CharSequenc
 	public static void writeJavaScriptUrl(final Response response, final CharSequence url,
 		final String id, boolean defer, String charset, boolean async)
 	{
-		response.write("<script type=\"text/javascript\" ");
+		response.write("<script ");
+		getNonceAttribute().ifPresent(response::write);
+		response.write("type=\"text/javascript\" ");
```

Comment:
There is no need to split line 155 into 156 and 158. The `nonce` attribute could be appended after the `type` attribute. This will save us one call to `Response#write()`